src/tesk_core/job.py

```diff
@@ -34,9 +39,15 @@ def run_to_completion(self, poll_interval, check_cancelled, pod_timeout):
                 raise ApiException(ex.status, ex.reason)
         is_all_pods_running = False
         status, is_all_pods_running = self.get_status(is_all_pods_running)
+        # notify the callback receiver that the job is running
```

Comment:
I think the code is obvious enough that it doesn't need a comment. I think the same goes for all other places where you call `callback.send()`, perhaps with the exception of this comment in the `taskmaster` module:

```
            # send "SYSTEM_ERROR" to callback receiver if taskmaster completes
            # but the output filer fails
```